packages/atomic/cypress.json

```diff
@@ -1,4 +1,5 @@
 {
+  "projectId": "5ph2j4",
```

Comment:
Could the `projectId` be `atomic`?

Reply:
the project id is generated by dashboard, so unfortunately we can't change it